.github/ISSUE_TEMPLATE/package.yml

```diff
@@ -1,16 +1,16 @@
 name: Packages
 description: Report problems and suggest ideas for the packages
-labels: [package]
+labels: [meta]
```

Comment:
The label `package` is still in use for npm/Packagist packages issues or PRs and it shouldn't be confused with `meta`, which is for the internal repo mangement, the build process, the lint process... etc. Why you've changed it?

Reply:
We can revert it - I just went by the descriptions :)

Reply:
We can add another template for meta describing what it is. Sometimes users open issues that fall into this category.